backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  enableAccessibilityFeatures Boolean                  @default(false) @map("enable_accessibility_features")
+  enableUtilitiesIncluded     Boolean                  @default(false) @map("enable_utilities_included")
+  amiChart                    AmiChart[]
+  multiselectQuestions        MultiselectQuestions[]
+  listings                    listings[]
+  reservedCommunityTypes      ReservedCommunityTypes[]
+  translations                Translations[]
+  user_accounts               UserAccounts[]
+
+  @@map("jurisdictions")
+}
+
+model ListingEvents {
+  id        String            @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt DateTime          @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt DateTime          @updatedAt @map("updated_at") @db.Timestamp(6)
+  type      MarketingTypeEnum
+  startDate DateTime?         @map("start_date") @db.Timestamptz(6)
+  startTime DateTime?         @map("start_time") @db.Timestamptz(6)
+  endTime   DateTime?         @map("end_time") @db.Timestamptz(6)
+  url       String?
+  note      String?
+  label     String?
+  listingId String?           @map("listing_id") @db.Uuid
+  fileId    String?           @map("file_id") @db.Uuid
+  assets    Assets?           @relation(fields: [fileId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings  listings?         @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@map("listing_events")
+}
+
+model ListingFeatures {
+  id                      String    @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt               DateTime  @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt               DateTime  @updatedAt @map("updated_at") @db.Timestamp(6)
+  elevator                Boolean?
+  wheelchairRamp          Boolean?  @map("wheelchair_ramp")
+  serviceAnimalsAllowed   Boolean?  @map("service_animals_allowed")
+  accessibleParking       Boolean?  @map("accessible_parking")
+  parkingOnSite           Boolean?  @map("parking_on_site")
+  inUnitWasherDryer       Boolean?  @map("in_unit_washer_dryer")
+  laundryInBuilding       Boolean?  @map("laundry_in_building")
+  barrierFreeEntrance     Boolean?  @map("barrier_free_entrance")
+  rollInShower            Boolean?  @map("roll_in_shower")
+  grabBars                Boolean?  @map("grab_bars")
+  heatingInUnit           Boolean?  @map("heating_in_unit")
+  acInUnit                Boolean?  @map("ac_in_unit")
+  hearing                 Boolean?
+  visual                  Boolean?
+  mobility                Boolean?
+  barrierFreeUnitEntrance Boolean?  @map("barrier_free_unit_entrance")
+  loweredLightSwitch      Boolean?  @map("lowered_light_switch")
+  barrierFreeBathroom     Boolean?  @map("barrier_free_bathroom")
+  wideDoorways            Boolean?  @map("wide_doorways")
+  loweredCabinets         Boolean?  @map("lowered_cabinets")
+  listings                listings?
```

Comment:
should this be related to a single listing?

Reply:
yeah so this is a confusing part of prisma but the actual guts of the relationship are defined on the listings model 

what is here is just the other side of it. Each one is only tied to 1 listing